stage1/init/init.go

```diff
@@ -612,10 +612,15 @@ func stage1() int {
 	// its peer group
 	// See https://www.kernel.org/doc/Documentation/filesystems/sharedsubtree.txt
 	if err := syscall.Mount("", "/", "none", syscall.MS_REC|syscall.MS_SLAVE, ""); err != nil {
-		log.Fatalf("error making / a slave mount: %v", err)
+		log.Fatalf("Error making / a slave mount: %v", err)
 	}
 	if err := syscall.Mount("", "/", "none", syscall.MS_REC|syscall.MS_SHARED, ""); err != nil {
-		log.Fatalf("error making / a shared and slave mount: %v", err)
+		log.Fatalf("Error making / a shared and slave mount: %v", err)
+	}
+
+	if err := mountHostCgroups(); err != nil {
+		log.Fatalf("Couldn't mount the host cgroups: %v\n", err)
```

Comment:
Is it ok if I keep the name but add a comment like

```
// mount host cgroups in the rkt mount namespace
if err := mountHostCgroups(); err != nil {
...
```

and I document the function?